lib/resque.rb

```diff
@@ -165,6 +165,12 @@ def prune_interval
     @prune_interval || DEFAULT_PRUNE_INTERVAL
   end
 
+  attr_writer :enqueue_front
+  def enqueue_front
+    return @enqueue_front unless @enqueue_front.nil?
+    @enqueue_front = false
+  end
```

Comment:
(In that case, I am not sure which approach I prefer. I might even lean towards the attr_accessor